modules/services/analytics/main.tf

```diff
@@ -180,7 +181,7 @@ resource "aws_instance" "analytics" {
   ami = var.analytics_image_id
   instance_type = var.analytics_instance_type
   vpc_security_group_ids = [aws_security_group.analytics.id]
-  subnet_id = tolist(data.aws_subnet_ids.default.ids)[0]
+  subnet_id = local.subnet_id
```

Comment:
What will happen to the existing `aws_instance.analytics` resources after this change? Won't this force their re-creation? If yes, we should add the following code to the bottom of this resource, to guarantee backward compatibility:
```terraform
  lifecycle {
    ignore_changes = [subnet_id]
  }
```

Reply:
@Agrendalath Good catch. Thanks for pointing this out. Suggested config is now added.